frontend/src/sdk/useAppConfiguration.ts

```diff
@@ -15,10 +15,11 @@ const useAppConfiguration = (): AppConfigurations | null => {
     if (auth && !appConfigurations) {
       setAppConfigurations({
         appFetch: commonFetch(auth),
-        wsAppSettings: async () => {
+        wsAppSettings: async (options: { wsPrefix?: string; pathPrefix?: string }) => {
+          const prefix = (options?.wsPrefix || options?.pathPrefix || '/wss/k8s') as string;
           const token = await auth.getToken();
           return {
-            host: `wss://${location.host}/wss/k8s`,
+            host: `wss://${location.host}${prefix.indexOf('/') === 0 ? '' : '/'}${prefix}`,
```

Comment:
FWIW seeing how we use this path in the URL - maybe we should rename `wsPrefix` to `wsPath` or similar. It's not really being used as a prefix.